src/company_cmd.cpp

```diff
@@ -553,7 +553,7 @@ Company *DoStartupNewCompany(bool is_ai, CompanyID company = INVALID_COMPANY)
 	ResetCompanyLivery(c);
 	_company_colours[c->index] = (Colours)c->colour;
 
-	c->money = c->current_loan = (100000ll * _economy.inflation_prices >> 16) / 50000 * 50000;
+	c->money = c->current_loan = (min(100000ll, _economy.max_loan) * _economy.inflation_prices >> 16) / 50000 * 50000;
```

Comment:
Can you put this in a variable like `initial_load` or something. Just more clear that way :)